hash/psx.xml

```diff
@@ -29414,6 +29414,31 @@ For an overview of US PS1 discs that have not been dumped and added to redump's
 		</part>
 	</software>
 
+	<software name="vribbon">
```

Comment:
Did you test this game in latest MAME?
1. Knowing that this specific game can disc swap with any Audio CD is already a `supported="partial"` in itself (i.e. without considering for now that is arguably the main selling point), 
2. nm, if it isn't playable at all `supported="no"`
NOTE: This is a rhythm game so beats syncing with music is again a liability.

Reply:
I had tested it in MAME but only the ability to load and take input; I'll go further in. Thank you! I'll withdraw and come back when it's more ready.